package.json

```diff
@@ -15,6 +15,7 @@
     "axios": "^1.7.2",
     "child_process": "^1.0.2",
     "fs-extra": "^11.2.0",
+    "libreoffice-convert": "^1.6.0",
```

Comment:
Might need to add this to the dependencies, so that if someone does not have "libreoffice" installed, it will automatically install.

This is in node-zerox/scripts/install-dependencies.js.
<img width="719" alt="Screen Shot 2024-08-27 at 10 45 25 AM" src="https://github.com/user-attachments/assets/98bc96fe-38aa-4ac6-878e-0af822803f6b">

It would probably look like the below (haven't tested this script yet):
```
 try {
      await execPromise("soffice --version");
    } catch {
      if (process.platform === "darwin") {
        await installPackage("brew install libreoffice", "LibreOffice");
      } else if (process.platform === "linux") {
        const command = sudoAvailable
          ? "sudo apt-get update && sudo apt-get install -y libreoffice"
          : "apt-get update && apt-get install -y libreoffice";
        await installPackage(command, "LibreOffice");
      } else {
        throw new Error(
          "Please install LibreOffice manually from https://www.libreoffice.org/download/download/"
        );
      }
    }
```

Reply:
verified and the script works (for both mac and linux in docker)